steps/bundler/0.0.3/step.yml

```diff
@@ -0,0 +1,37 @@
+title: Bundler
+summary: Install Ruby Dependencies with Bundler
+description: Install Ruby Dependencies with Bundler
+website: https://github.com/chrisfuller/bitrise-step-bundler
+source_code_url: https://github.com/chrisfuller/bitrise-step-bundler
+support_url: https://github.com/chrisfuller/bitrise-step-bundler/issues
+published_at: 2019-09-08T16:41:15.363368-07:00
+source:
+  git: https://github.com/chrisfuller/bitrise-step-bundler.git
+  commit: 5941e841a021216b90c13e32f1b38bfdc7ac6691
+host_os_tags:
+- osx
+- ubuntu
+type_tags:
+- utility
+- dependency
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- bundle_install_jobs: "4"
+  opts:
+    description: |
+      The maximum number of parallel download and install jobs.
+    is_expand: true
+    is_required: true
+    title: --jobs
```

Comment:
The underlying flag is not the best as step input title, as may not all of the Bitrise users know what is it for, `The maximum number of parallel jobs` sounds better, we usually mention the underlying flag in the input's description.

Reply:
Fixed in `v0.0.4` 👍